### PR TITLE
Update Work history section, move Personal statement up and remove Unpaid experience

### PR DESCRIPTION
**Pull request by frankieroberto**
This makes the following changes to the Application:

* Moves the Personal statement section up, above the References section (as this is more likely to be a factor in the initial application sift)
* Updates the Work history section to remove job descriptions (as these may no longer be asked for).
* Displays a summary table in the Work history section if the candidate has no work history, to show exactly how they answered (in line with the recent changes to this page).
* Comments-out the "Unpaid experience and volunteering" section, for testing purposes, as this may be removed from the application, with the intention that candidates give this information either within the work history (for longer unpaid roles) and/or within their personal statement.

## Screenshots

Screenshot 2020-12-10 at 10 58 43

Screenshot 2020-12-10 at 10 58 34

### CODE REVIEW

**Comment by adamsilver** [comment]
Can we just hide it with a comment to save fishing it back out later?

**Comment by frankieroberto** [comment]
@adamsilver I wasn't planning on merging this until we've tested it and made a decision, so hopefully we'd be ok to remove it at that point (if we go ahead with this)?

**Comment by frankieroberto** [comment]
@adamsilver I've updated the prototype so that 1 in 3 of the applications have this preselected reason for the lack of work history (from selecting the radio option):

Screenshot 2020-12-03 at 16 08 10

...and another 1 in 3 have this reason, which they would have entered themselves:

Screenshot 2020-12-03 at 16 07 59

Kept the design very basic for now, as I know you're looking at this page anyway.

File: scripts/generate-applications.js [hunk 1]
```diff
@@ -80,7 +80,9 @@ const generateFakeApplication = (params = {}) => {
     events,
     personalDetails,
     contactDetails: params.contactDetails || generateContactDetails(faker, personalDetails),
-    interviewNeeds: generateInterviewNeeds(),
+    interviewNeeds: generateInterviewNeeds(faker),
+    workHistoryAnswer: faker.helpers.randomize(['yes', 'no', 'full-time-education']),
```
Comment (adamsilver) [inline scripts/generate-applications.js:84]:
```suggestion
    workHistoryAnswer: faker.helpers.randomize(['yes', 'no--work-history', 'no--full-time-education']),
```

**Pull request merged**